src/tracks/SampleTrack.cpp

```diff
@@ -562,10 +565,12 @@ SampleTrack::SampleTrack( TrackContainer* tc ) :
 							tr( "Volume" ) ),
 	m_panningModel( DefaultPanning, PanningLeft, PanningRight, 0.1f,
 					this, tr( "Panning" ) ),
+	m_effectChannelModel( 0, 0, 0, this, tr( "FX channel" ) ),
 	m_audioPort( tr( "Sample track" ), true, &m_volumeModel, &m_panningModel, &m_mutedModel )
 {
 	setName( tr( "Sample track" ) );
 	m_panningModel.setCenterValue( DefaultPanning );
```

Comment:
After this line add `m_effectChannelModel.setRange( 0, Engine::fxMixer()->numChannels()-1, 1);`.
This fixes problem with changing FX channel for newly added sample tracks.